lib/simpleofflinecache.js

```diff
@@ -5,9 +5,32 @@ var cacheHelper = require('sw-cache-helper');
 var debug = 0 ? console.log.bind(console, '[SimpleOfflineCache]') :
  function(){};
 
-function SimpleOfflineCache(cacheName) {
+// Default Match options, not exposed.
+var DEFAULT_MATCH_OPTIONS = {
+  ignoreSearch: false,
+  ignoreMethod: false,
+  ignoreVary: false
+};
+// List of different policies
+var MISS_POLICIES = [
+  'fetchAndCache'
+];
+
+
+/**
+ * Constructor for the middleware that serves the content of a
+ * cache specified by it's name.
+ * @param {string} cacheName Name of the cache that will be serving the content
+ * @param {object} [options] Object use to setup the cache matching alternatives
+ * @param {string} [missPolicy] Name of the policy to follow if a request miss
+ *                 when hitting the cache.
+ */
+function SimpleOfflineCache(cacheName, options, missPolicy) {
   this.cacheName = cacheName || cacheHelper.defaultCacheName;
   this.cache = null;
+  this.options = options || DEFAULT_MATCH_OPTIONS;
+  this.missPolicy = MISS_POLICIES.indexOf(missPolicy) !== -1 ?
+    missPolicy : 'fetchAndCache';
```

Comment:
Maybe we should warn in the console. Imagine you misspelled the policy. Maybe you are choosing one completely contrary to `fetchAndCache`.